docs/architecture/baking.md

```diff
@@ -9,7 +9,7 @@ Baking is the process of creating new blocks in the Tezos blockchain.
 Bakers are daemons complementary to Tezos nodes that cooperate to achieve consensus about the next block to add.
```

Comment:
We don't use _daemon_ in Octez. We say baking software/binaries/executables.

Technically speaking the baker is not a _daemon_, as it does not run in the background (and the user can C-c / C-z it). You can run it as a service, but the baker is not ...sshd

Reply:
replaced by executables

Reply:
I'm not sure it's so important. We do use "daemons" in many places in the Octez doc (see <https://tezos.gitlab.io/search.html?q=daemon#>), but I changed it here to be more accessible.